sycl/test-e2e/Basic/accessor/accessor.cpp

```diff
@@ -818,20 +818,18 @@ int main() {
     }
   }
 
-  // SYCL2020 4.9.4.1: calling require() on empty accessor should throw
+  // SYCL2020 4.9.4.1: calling require() on empty accessor should not throw.
```

Comment:
I'm confused. This is exactly wrong. The spec at 4.9.4.1 table 131 first entry says:
```Throws exception with the
errc::invalid error code if
(acc.empty() == true).
```

Reply:
https://github.com/KhronosGroup/SYCL-Docs/pull/434/files in this PR, it was updated to say it's allowed.